bin/cli.js

```diff
@@ -420,9 +422,14 @@ else if (scripts.length === 0) {
   runStream(process.stdin, process.stdout, mode, parenthesis);
 }
 else {
-  //work through the queue of scripts
-  scripts.forEach(function (arg) {
-    // run a script file
-    runStream(fs.createReadStream(arg), process.stdout, mode, parenthesis);
-  });
+  fs.stat(scripts[0], (e, f) => {
```

Comment:
Can you please replace the arrow function with a regular function so the cli also works with old version of node.js?